lib/client/js/ui/base/Control.js

```diff
@@ -10,17 +10,65 @@ var Control = (function () {
 
     /**
      * @alias Control
+     * @param {Object} config - The configuration of the control
+     * @property {Store} config.store - the props store
      */
-    function Control() {
+    function Control(config) {
         /** @member {number} */
         /** @member {Object} */
         /** @member {String} */
         this._id = idCounter++;
         this._$ = null;
         this._namespace = null;
+
+        if (config && config.store) {
+            this._store = store;
+        }
     }
 
     /**
+     * Start State Management
+     */
+
+    /**
+     * @public
+     * @function
+     * @param {String} name - the name of the prop to set in the store
+     * @param {*} value - the value of the prop to set in the store
```

Comment:
We wouldn't need the `@param` documentation here, right? Since the Store class has this documentation already.